Sources/ArcGISToolkit/Components/UtilityNetworkTrace/UtilityNetworkTraceViewModelTrace.swift

```diff
@@ -82,19 +82,21 @@ extension UtilityNetworkTraceViewModel {
         
         /// The extent of the trace's geometry result with a small added buffer.
         var resultExtent: Envelope? {
-            if let resultEnvelope = GeometryEngine.combineExtents(of: [
+            let geometries = [
                 utilityGeometryTraceResult?.multipoint,
                 utilityGeometryTraceResult?.polygon,
                 utilityGeometryTraceResult?.polyline
-            ].compactMap { $0 }),
-               let expandedEnvelope = GeometryEngine.buffer(
-                around: resultEnvelope,
-                distance: 200
-               ) {
-                return expandedEnvelope.extent
-            } else {
+            ]
+                .compactMap { $0 }
+                .filter { !$0.isEmpty }
+            
+            guard !geometries.isEmpty,
```

Comment:
Ideally this check shouldn't be necessary; the geometry engine method should just return `nil` (which it does) and not log anything (which it doesn't).

Reply:
Agreed. I think this would be covered by Swift 3246, no? Any changes needed in this PR here?

Reply:
It is likely that this will be addressed in the near future. Might be best to wait and see.

Reply:
@philium With 3264 merged, I no longer seem to be able to trigger the empty geometries error. I can, however, easily trigger the "The input feature does not belong to this utility network." log. Is that expected after 3264 still?

Reply:
Yes, it is expected, at least for now.